README.md

```diff
@@ -1998,7 +1998,9 @@ Other Style Guides
 ## Control Statements
 
   <a name="control-statements"></a>
-  - [17.1](#control-statements) In case your control statement (`if`, `while` etc.) gets too long or exceeds the maximum line length, each (grouped) condition could be put into a new line. It’s up to you whether the logical operator should begin or end the line.
+  - [17.1](#control-statements) In case your control statement (`if`, `while` etc.) gets too long or exceeds the maximum line length, each (grouped) condition could be put into a new line. The logical operator should begin the line.
+
+  > Why? This ensures readability.
```

Comment:
Most of the rules in the style guide are for readability. Can we make this a little more useful? e.g. why beginning of line vs end of line? Mostly to be consistent with other styles, like how we chain with dots at the beginning of the line.

Reply:
(Definitely to be consistent, but also to enforce only one way to do it)